stock_barcodes/wizard/stock_barcodes_read_inventory_views.xml

```diff
@@ -29,12 +29,6 @@
                                             <field name="location_ids" />
                                         </div>
                                     </div>
-                                    <div class="row">
```

Comment:
Why removing this?

Reply:
Not very practical. When there are many references it takes to much space in the screen

Reply:
Ok, but then do a separate commit with proper commit message explaining that or at least comment it in the same commit